dashconverter.py

```diff
@@ -89,21 +89,23 @@ def widget_transform(cls):
 	def convert_s2t(cls, widgets):
 		# Function to convert Screenboard to Timeboard.
 		# Takes the widgets as input and output the widgets properly formatted.
-		# Appens an additionnal attribute for the hostmap.
+		# Appends an additional attribute for the hostmap.
 		# no output, just tranforms the cls.graphs
-		
+		# If "show a title" is unchecked, use the query as the new widget title.
+
 		for i in range(len(widgets)):
 
 			if 'tile_def' in widgets[i]:
 				if 'conditional_formats' not in widgets[i]['tile_def']['requests'][0]:
 					widgets[i]['tile_def']['requests'][0]['conditional_formats'] = []
-			else: 
+			else:
 				widgets[i]['tile_def'] = 'outdated'
 				print "One of the widgets' type is outdated and won't be ported.\n To solve this, just click on edit the dashboard, open a widget, hit done and save the dashboard.\n Then run the script again."
-				print widgets[i]
 
-			if widgets[i]['type'] == 'hostmap':
+			if not (('title_text' in widgets[i]) and (isinstance(widgets[i]['title_text'],str))):
```

Comment:
Checks if the title_text property exists in widgets[i] and checks if the value is a valid string if it does exists.
If not, create/set the widget's title_text property to the query e.g. title_text = "avg:system.cpu.user{*}"